module.json

```diff
@@ -19,8 +19,8 @@
     "allowBugReporter": true
   },
   "version": "1.13.0",
-  "minimumCoreVersion": "9",
```

Comment:
Is it practical to implement this feature on such a way that it does not initialize in v9 and this we do not need to bump this?

Reply:
Sure. It just requires a bit of extra code